deployment/Makefile

```diff
@@ -174,6 +181,34 @@ indicator-worker-logs:
 	@echo "------------------------------------------------------------------"
 	@docker-compose -p $(PROJECT_ID) logs -f --tail=50 indicator-worker
 
+realtime-worker:
+	@echo
```

Comment:
@ismailsunni this section is useful to start realtime worker independently, in case we use prebuilt docker image
It is better to create another one for headless

Reply:
Noted.